src/main/java/se/sundsvall/byggrintegrator/service/ApiResponseMapper.java

```diff
@@ -0,0 +1,27 @@
+package se.sundsvall.byggrintegrator.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import se.sundsvall.byggrintegrator.api.model.KeyValue;
+import se.sundsvall.byggrintegrator.model.NeighborhoodNotificationsDto;
+
+@Component
+public class ApiResponseMapper {
+
+	public List<KeyValue> mapToKeyValueResponseList(List<NeighborhoodNotificationsDto> errands) {
+		List<KeyValue> response = new ArrayList<>();
+
+		errands.forEach(
```

Comment:
Cover for possible NPE: `Optional.ofNullable(errands).orElse(Collections.emptyList()).forEach(`?